frontend/src/components/search/components/common/filter-select/filter-select.tsx

```diff
@@ -0,0 +1,88 @@
+import { FC } from 'common/types/types';
+import { useId, useState, useEffect } from 'hooks/hooks';
+import { IconName } from 'common/enums/components';
+import { Icon } from 'components/common/common';
+
+import styles from './styles.module.scss';
+
+type Props = {
+  activeFilterId: string;
+  onChangeFilterId: (value: string) => void;
+  defaultFilterId: string;
+  title: string;
+  options: {
+    text: string;
+    value: string;
+  }[];
+  toggleAllFilters: boolean;
+  onChangeToggleAllFilters: (toggle: boolean) => void;
+};
+
+const FilterSelect: FC<Props> = ({
+  activeFilterId,
+  onChangeFilterId,
+  defaultFilterId,
+  title,
+  options,
+  toggleAllFilters,
+  onChangeToggleAllFilters,
+}) => {
+  const [filterTitle, setFilterTitle] = useState(title);
```

Comment:
Do you really need this state? Looks like you can get it in wrap component using active filter id?